client/layout/index.jsx

```diff
@@ -45,17 +45,16 @@ import { withCurrentRoute } from 'calypso/components/route';
 import QueryExperiments from 'calypso/components/data/query-experiments';
 import Experiment from 'calypso/components/experiment';
 import QueryReaderTeams from 'calypso/components/data/query-reader-teams';
```

Comment:
In that case we can remove `QueryReaderTeams` too

Reply:
It's still needed, or the teams won't be populated in the new `isNavUnificationEnabled` selector.

Reply:
Yeah, I meant moving it to a more appropriate place together with `getReaderTeams`.